labgrid/protocol/OneWireProtocol.py

```diff
@@ -0,0 +1,15 @@
+import abc
+
+
+class OneWireProtocol(abc.ABC):
```

Comment:
At the protocol level, this should be more generic. Maybe call it DigitalOutput instead?